frontend/appflowy_flutter/lib/plugins/database_view/calendar/presentation/calendar_page.dart

```diff
@@ -367,9 +372,16 @@ void showEventDetails({
 }
 
 class UnscheduledEventsButton extends StatefulWidget {
-  final DatabaseController databaseController;
+  const UnscheduledEventsButton({
+    super.key,
+    required this.databaseController,
+    required this.rowCache,
+    required this.viewId,
+  });
 
-  const UnscheduledEventsButton({super.key, required this.databaseController});
+  final DatabaseController databaseController;
+  final RowCache rowCache;
+  final String viewId;
```

Comment:
Both of these are in database controller already. Up to you if you want to add it as well.